src/auth/roles/index.ts

```diff
@@ -129,6 +129,30 @@ const rolesInitialiser: RolesInitialiser = (client) => {
       const result = await client.raw('SELECT EXISTS ?', [inner]);
       return result.rows[0];
     },
+    getUserRole: async ({ userId, organisationId }) => {
```

Comment:
If we call the object `Roles`, this method name can be simplified.

```js
Roles.fromUser(user, org?)
```

Reply:
I think this too easily confused with the `Role` enum. I think we should think carefully about a convention for naming enums and modules to implement. Currently I am using:
- singular, capitalised for `enums`
- appending `initialiser`/`interface` for the various states of a module

Happy for suggestions of a better naming convention.